crates/ruff/src/rules/pylint/rules/subprocess_run_without_check.rs

```diff
@@ -56,10 +62,21 @@ pub(crate) fn subprocess_run_without_check(checker: &mut Checker, call: &ast::Ex
         .is_some_and(|call_path| matches!(call_path.as_slice(), ["subprocess", "run"]))
     {
         if call.arguments.find_keyword("check").is_none() {
-            checker.diagnostics.push(Diagnostic::new(
-                SubprocessRunWithoutCheck,
-                call.func.range(),
-            ));
+            let mut diagnostic = Diagnostic::new(SubprocessRunWithoutCheck, call.func.range());
+            let text: &str = checker.locator().slice(call.range());
```

Comment:
Nit: the fix logic (and `diagnostic.set_fix` should be wrapped in a call to `if checker.patch(diagnostic.kind.rule()) { ... }`, to ensure that we skip the fix if the user marks the rule as unfixable.